pdfviewer/projects/pdfViewers/src/svypdfJsViewer.ts

```diff
@@ -107,7 +108,13 @@ export class SvyPdfJsViewer extends ServoyBaseComponent<HTMLDivElement> {
                 else this.disableTooltips();
                 this.fillOutFormFields();
                 this.hideFieldControls();
-            })
+            });
+            viewer.eventBus.on("pagechanging", (evt) => {
+                if(this.onPageChangedMethodID) {
```

Comment:
@SteveHawes  can you change this code that the if is outside of the viewer.eventBus?
because if there is no onPageChangeMethodID then we don't need to register a listener.
also i would rename the onPageChangeMethodID  to onPageChange 
that "methodid" is an old notation. we don't need that

Reply:
The code has been modified as requested and pushed.

Thanks